tdrs-frontend/.env.development

```diff
@@ -3,6 +3,8 @@
 # This file is loaded when running `npm start`
 #
 
+REACT_APP_BACKEND_HOST=http://localhost:8080
```

Comment:
Can you also add this to the base `.env` file? We may consider deleting this file in the future since nothing needs to change from the base.